src/components/Footer.tsx

```diff
@@ -0,0 +1,85 @@
+import classNames from 'classnames';
+import React, { useState } from 'react';
+
+type Props = {
+  count: number,
+  setFilter: (option: string) => void;
+  clearCompleted: () => void;
+  isCompleted: boolean,
+};
+
+export const Footer: React.FC<Props> = (
+  {
+    count,
+    setFilter,
+    clearCompleted,
+    isCompleted,
+  },
+) => {
+  const [isSelectedAll, setSelected] = useState(false);
```

Comment:
Consider using more descriptive names for the state variables and their setters, e.g., 'isSelectedAll' -> 'isAllSelected', 'setSelected' -> 'setAllSelected', etc.